src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerAgreementController.cs

```diff
@@ -327,5 +310,12 @@ public async Task<ActionResult> WhenDoYouWantToView(int? choice, string agreemen
                 }
             }
         }
+
+        [HttpGet]
+        [Route("agreements/{accountLegalEntityHashedId}/viewallagreements")]
+        public ActionResult ViewAllAgreements(string hashedAccountId, string accountLegalEntityHashedId)
+        {
+           throw new NotImplementedException();
```

Comment:
In this a placeholder for Vas's story?

Reply:
If you can call this it shouldn't throw it should put up a message - so this story could be shipped independently

Reply:
Ok has been reverted to previous functionality